style.css

```diff
@@ -0,0 +1,85 @@
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto&display=swap');
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  padding: 0;
+  margin: 0;
+  font-family: Georgia, 'Times New Roman', Times, serif;
+}
+
+.tool-bar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  flex-wrap: wrap;
+  background-color: #3c3a39;
+  height: 48px;
+  position: fixed;
+  top: 0;
+  right: 0;
+  left: 0;
+}
+
+.tool-bar a {
+  color: rgb(187, 171, 171);
+  font-family: 'Roboto', sans-serif;
+  padding: 12px 24px;
+  text-decoration: none;
+}
+
+.fas {
+  color: #ffff;
+  padding: 19px 36px;
+}
+
+.container {
+  margin-top: 48px;
+  background-image: url(../media/background.png);
```

Comment:
The background image for the headline section has the wrong placement. Kindly see the screenshot below and use the design page to fix the issues.

Reply:
I'm sorry to inform you, I didn't find the screenshot here.

Reply:
Thank you for pointing it out! Everything is fine here. Happy coding!